client/my-sites/domains/domain-management/controller.jsx

```diff
@@ -18,7 +16,20 @@ import DomainManagementData from 'components/data/domain-management';
 import EmailData from 'components/data/domain-management/email';
 import EmailForwardingData from 'components/data/domain-management/email-forwarding';
 import NameserversData from 'components/data/domain-management/nameservers';
-import paths from 'my-sites/domains/paths';
+import {
```

Comment:
this file would have has such a smaller diff if we had written:
`import * as paths from 'my-sites/domains/paths';`

:/

Reply:
see my response above. smaller diff but I _think_ that the changes mean we will get smaller bundles once we stop compiling to CommonJS modules